Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -1070,18 +1430,103 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
 
                             # Create Alert Rule Resource Object
                             $newAnalyticRule = [PSCustomObject]@{
-                                type       = "Microsoft.OperationalInsights/workspaces/providers/alertRules";
+                                type       = $contentToImport.TemplateSpec ? "Microsoft.OperationalInsights/workspaces/providers/AlertRuleTemplates" : "Microsoft.OperationalInsights/workspaces/providers/alertRules";
```

Comment:
analytic rule type is not correct. This should be "Microsoft.SecurityInsights/AlertRuleTemplates"